src/single.rs

```diff
@@ -160,6 +160,14 @@ impl Quaint {
         Ok(Self { inner, connection_info })
     }
 
+    #[cfg(feature = "sqlite")]
+    pub fn new_in_memory() -> crate::Result<Quaint> {
```

Comment:
I was first saying hell no, but now when I think about this, actually it makes sense. The only problem is you can't pass connection params through this interface.

Reply:
What params would you pass?

Reply:
Note: if we go with my last suggestion on the other discussion, we should pass the attached name here.

Reply:
(optionally)